examples/ColorPicker.advancedPopover.tsx

```diff
@@ -69,7 +69,10 @@ export default () => {
           visible={isOpen}
           placement='bottom-start'
         >
-          <IconButton onClick={() => setIsOpen((open) => !open)}>
+          <IconButton
+            label='Color Picker toggle'
```

Comment:
Would a better label be "Show/Hide Color Picker" or "Color Picker Toggle Visibility"? Because just "toggle" could also mean toggle the *color format (e.g. hsla, hex)* and not the *visibility* of the color picker

Reply:
Fixed!

Reply:
can we use more natural language?

i.e. "Toggle color picker visibility"